Add leader election configuration instructions (#1359)
rancher/rancher-docs: docs/troubleshooting/other-troubleshooting-tips/rancher-ha.md

NAME                 HOLDER                    AGE
cattle-controllers   rancher-dbc7ff869-gvg6k   6h10m
```

#### Configuration

_Available as of Rancher 2.8.3_

> If the Kubernetes API experiences latency, the Rancher replica holding the leader lock may not be able to renew the lease before the lease becomes invalid.
> To mitigate this, you can set environment variables in the `rancher` Deployment to modify the default parameters for leader election:

Reviewer: ```suggestion
If the Kubernetes API experiences latency, the Rancher replica holding the leader lock may not be able to renew the lease before the lease becomes invalid. To mitigate this, you can set environment variables in the `rancher` Deployment to modify the default parameters for leader election:
```

> _Available as of Rancher 2.8.3_
> 
> In the unfortunate event of Kubernetes API latency degradation, the Rancher replica holding the leader lock may not be able to renew the lease within its validity duration.
> In order to mitigate this, it is possible to modify the default parameters for leader election using environment variables:

Reviewer: ```suggestion
If the Kubernetes API experiences latency, the Rancher replica holding the leader lock may not be able to renew the lease before the lease becomes invalid. To mitigate this, in Rancher v2.8.3 and later you can use environment variables to modify the default parameters for leader election:
```

Reviewer 2: I think it's preferable for the version availability information to be highly visible instead of appearing in the middle of a paragraph where it can missed.

Author: +1, BTW I took the example from:
https://ranchermanager.docs.rancher.com/reference-guides/user-settings/user-preferences#confirmation-setting

I can apply the suggested change while keeping the banner.

> - `CATTLE_ELECTION_LEASE_DURATION` configures the [lease duration](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection#LeaderElectionConfig.LeaseDuration) (defaults to 45s)
> - `CATTLE_ELECTION_RENEW_DEADLINE` configures the [renew deadline](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection#LeaderElectionConfig.RenewDeadline) (defaults to 30s)
> - `CATTLE_ELECTION_RETRY_PERIOD` configures the [retry period](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection#LeaderElectionConfig.RetryPeriod) (defaults to 2s)

Reviewer: ```suggestion
- `CATTLE_ELECTION_LEASE_DURATION`: The [lease duration](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection#LeaderElectionConfig.LeaseDuration). The default value is 45s.
- `CATTLE_ELECTION_RENEW_DEADLINE`: The [renew deadline](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection#LeaderElectionConfig.RenewDeadline). The default value is 30s.
- `CATTLE_ELECTION_RETRY_PERIOD`: The [retry period](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection#LeaderElectionConfig.RetryPeriod). The default value is 2s.
```
